tekton/images/test-runner/runner.sh

```diff
@@ -89,7 +89,36 @@ export SOURCE_DATE_EPOCH
 
 # actually start bootstrap and the job
 set -o xtrace
+
```

Comment:
NIT: Perhaps we could have an heading echo, like "=== Running ./runner.sh backward compatibility test runner  === "